src/time/time.cr

```diff
@@ -1,3 +1,50 @@
+# Time library allows you to inspect, analyze, and calculate time. Here are some examples:
+# 
+# ### Basic Usage
+# 
+#     time = Time.now
+#     #=> 2016-02-15 10:20:30 UTC
+#     
+#     time.year    #=> 2015
+#     time.month   #=> 2
+#     time.day     #=> 15
+#     time.hour    #=> 10
+#     time.minute  #=> 20
+#     time.second  #=> 30
+# 
+#     # Creating a time instance
+#     Time.new(2016,2,15)
+#     #=> 2016-02-15 00:00:00
+# 
+#     # Specifying time & timezone
```

Comment:
Well, not exactly a timezone yet, just UTC vs localtime, though the latter is kinda broken, see #1521